client/src/pages/ChatPage.js

```diff
@@ -32,13 +32,16 @@ class ChatPage extends Component {
             return <p>no messages yet! {JSON.stringify(this.props)}</p>
         }
     }
-
     render() {
         return (
             <Container fluid className="vh-100 text-center" >
                 <Row className="h-10 bg-dark text-light sticky-top">
                     <Col>
-                        <h3>Chatr</h3>
+                        <h1>Chatr</h1>
+                        <Button variant="outline-danger" size="sm" style={{float: "right", marginTop: 0, marginBottom: "5px"}} type="submit" onClick={(e)=>{
+                            localStorage.setItem('login', "")
+                            this.props.loginUpdater(false)
+                        }}>Log Out</Button>
```

Comment:
I think there is a missing preventDefault in the onClick for the send button. I thought I had that in my version but I guess it's not there anymore

Reply:
This has been fixed